jasmine/spec/inverted-index-test_spec.js

```diff
@@ -23,20 +24,38 @@ describe('Read book data', () => {
   it('should be a valid json file', () => {
     expect(index.isJson()).toBe(true);
   });
+
+  it('should have a tile and text field', () => {
+    index.createIndex('invalid2.json', invalid)
```

Comment:
### Critical 

@andela-wmaina you might want to listen to HoundCI though, please fix!